cloud.foundry.cli/src/main/java/cloud/foundry/cli/logic/apply/SpaceDevelopersRequestsPlaner.java

```diff
@@ -22,7 +22,7 @@
     /**
      * Creates the requests to assign/revoke space developer's permission.
      *
-     * @param spaceDevelopersOperations the SpaceDevelopersOperations object.
+     * @param spaceDevelopersOperations operations instance that is used to manipulate the space developersn
```

Comment:
Unfortunately you had a typo here:
```suggestion
     * @param spaceDevelopersOperations operations instance that is used to manipulate the space developers
```